DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/TopCourses/Index.cshtml

```diff
@@ -0,0 +1,80 @@
+﻿@inject IConfiguration configuration
+
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Centre.TopCourses
+@using Microsoft.Extensions.Configuration
+@model TopCoursesViewModel
+
+
+@{
+  ViewData["Title"] = "Top courses";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
+  ViewData["HeaderPathName"] = "Tracking System";
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+@section NavBreadcrumbs {
+  <partial name="~/Views/TrackingSystem/Shared/_CentreDashboardBreadcrumbs.cshtml"/>
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 class="nhsuk-heading-xl">Top courses</h1>
+
+    <table role="table" class="nhsuk-table-responsive">
+      <thead role="rowgroup" class="nhsuk-table__head">
+      <tr role="row">
+        <th role="columnheader" class="nhsuk-u-padding-left-2" scope="col">
+          Name
+        </th>
+        <th role="columnheader" class="nhsuk-u-padding-left-2" scope="col">
+          Delegates
+        </th>
+        <th role="columnheader" class="nhsuk-u-padding-left-2" scope="col">
+          In progress
+        </th>
+        <th role="columnheader" class="nhsuk-u-padding-left-2" scope="col">
+          Completed
+        </th>
+        <th role="columnheader" class="nhsuk-u-padding-left-2" scope="col">
+          Pass rate
+        </th>

```

Comment:
This was called "Pass ratio" in the old site and in the ticket description, but I decided to call it "Pass rate" instead as they are displayed as percentages, so I thought this made more sense. Happy to change it back to "Pass ratio" if we want.

Reply:
It was decided to use "Pass rate" instead of "Pass ratio" during Show & Tell on 02/07/2021. Ticket description changed accordingly.